misc/xml-to-record-converter/src/test/resources/ballerina/sample_28.bal

```diff
@@ -0,0 +1,48 @@
+@xmldata:Namespace {prefix: "n", uri: "[www.example.org/personName](http://www.example.org/personName)"}
+type N_Name record {
+    string __text;
+    @xmldata:Attribute
+    string n = "[www.example.org/personName](http://www.example.org/personName)";
+};
+
+@xmldata:Namespace {prefix: "c", uri: "[www.example.org/addressCity](http://www.example.org/addressCity)"}
+type C_City record {
+    string __text;
+    @xmldata:Attribute
+    string c = "[www.example.org/addressCity](http://www.example.org/addressCity)";
```

Comment:
We don't have to add the prefix to the fields according to the new implementation

Reply:
Fixed with [bfd1a23](https://github.com/ballerina-platform/ballerina-lang/pull/42699/commits/bfd1a239a8993c12bca58190302884efbea31f82)